src/main/java/net/minecraftforge/common/data/ForgeLootTableProvider.java

```diff
@@ -0,0 +1,162 @@
+package net.minecraftforge.common.data;
+
+import com.mojang.datafixers.util.Pair;
+import net.minecraft.advancements.criterion.ItemPredicate;
+import net.minecraft.data.DataGenerator;
+import net.minecraft.data.LootTableProvider;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+import net.minecraft.tags.Tag;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.storage.loot.*;
+import net.minecraft.world.storage.loot.conditions.Alternative;
+import net.minecraft.world.storage.loot.conditions.ILootCondition;
+import net.minecraft.world.storage.loot.conditions.Inverted;
+import net.minecraft.world.storage.loot.conditions.MatchTool;
+import net.minecraftforge.common.Tags;
+
+import java.lang.reflect.Field;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+public class ForgeLootTableProvider extends LootTableProvider {
```

Comment:
Should probably note that this is used for shears only right now

Reply:
Added comment